query/stdlib/influxdata/influxdb/rules.go

```diff
@@ -35,3 +37,78 @@ func (rule PushDownRangeRule) Rewrite(node plan.Node) (plan.Node, bool, error) {
 		Bounds:   rangeSpec.Bounds,
 	}), true, nil
 }
+
+// MergeFromFilterRule is a rule that pushes filters into from procedures to be evaluated in the storage layer.
```

Comment:
This comment can be removed now.

Reply:
It seems I forgot to rename it. Usually the refactors take care of that, but I think I manually changed the name.